dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Controllers/PatientController.cs

```diff
@@ -90,7 +118,26 @@ public class PatientController : Controller
 
             var responseContent = await ExecuteRestRequest(personId, recordId, uriBuilder.Uri, HttpMethod.Get, "2.0-preview");
 
-            return SafeJsonConvert.DeserializeObject<TimelineResponse>(responseContent, GetDeserializationSettings());
+            return SafeJsonConvert.DeserializeObject<TimelineResponse>(responseContent, _deserializerSettings);
+        }
+
+        private async Task PostTaskTracking(Guid personId, Guid recordId, Guid taskId, DateTimeOffset trackingDateTime)
```

Comment:
I haven't worked on the samples at all but isn't there a better way than to keep passing in the person and the record Id for each request?